dev/.rat-excludes

```diff
@@ -115,3 +115,5 @@ structured-streaming/*
 kafka-source-initial-offset-version-2.1.0.bin
 kafka-source-initial-offset-future-version.bin
 vote.tmpl
+SessionManager.java
```

Comment:
I guess we can't narrow this to the particular file in its particular directory? it's probably fine if not

Reply:
Yes.  Please see [RAT-240](https://issues.apache.org/jira/browse/RAT-240) CLI: RAT excludes file does not allow to set a full path